lib/index.ts

```diff
@@ -86,9 +94,26 @@ class PWMetrics {
         await sheets.appendResults(results.runs);
       }
     }
+
+    if (resultHasExpectationErrors && this.flags.expectations) {
+      throw new Error(messages.getMessage('HAS_EXPECTATION_ERRORS'));
+    }
+
     return results;
   }
 
+  resultHasExpectationErrors(metrics: MetricsResults): boolean {
+    const expectationErrors = metrics.timings.filter((timing: Timing) => {
+      const expectation = this.expectations[timing.id];
+      if (!expectation) {
+        return false;
+      }
+      const expectedErrorLimit = expectation.error;
+      return expectedErrorLimit !== undefined && timing.timing >= expectedErrorLimit;
```

Comment:
I've got it... just seems weird that you can set a threshold if an operator that will be ignored. Not a problem of this PR anyway..